DrivewayDentDeletion/Operators/cicd-webhook-triggers.yaml

```diff
@@ -507,96 +104,184 @@ spec:
       resources:
         inputs:
           - name: git-source
             resource: git-source
-    - name: ace-bernie-build
+    - name: build-ace-int-server-ace-api
+      # runAfter:
+      #   - build-mq
       taskRef:
-        name: build-push-ace-bernie-docker-image
+        name: build-push-docker-image
       params:
         - name: imageTag
           value: "$(params.imageTag)"
+        - name: imageName
+          value: "ace-api"
+        - name: dockerfile
+          value: "ACE-API.Dockerfile"
+        - name: contextPath
+          value: "Bar_files/ace-api"
       resources:
         inputs:
           - name: git-source
             resource: git-source
-    - name: ace-chris-build
+    - name: build-ace-int-server-ace-acme
+      runAfter:
+        - build-ace-int-server-ace-api
       taskRef:
-        name: build-push-ace-chris-docker-image
+        name: build-push-docker-image
       params:
         - name: imageTag
           value: "$(params.imageTag)"
+        - name: imageName
+          value: "ace-acme"
+        - name: dockerfile
+          value: "ACE-Acme.Dockerfile"
+        - name: contextPath
+          value: "Bar_files/ace-acme"
       resources:
         inputs:
           - name: git-source
             resource: git-source
-    - name: mq-build
+    - name: build-ace-int-server-ace-bernie
+      # runAfter:
+      #   - build-ace-int-server-ace-chris
       taskRef:
-        name: build-push-mq-docker-image
+        name: build-push-docker-image
       params:
         - name: imageTag
           value: "$(params.imageTag)"
+        - name: imageName
+          value: "ace-bernie"
+        - name: dockerfile
+          value: "ACE-Bernie.Dockerfile"
+        - name: contextPath
+          value: "Bar_files/ace-bernie"
       resources:
         inputs:
           - name: git-source
             resource: git-source
-    - name: ace-api-helm-deploy
+    - name: build-ace-int-server-ace-chris
       runAfter:
-        - ace-api-build
+        - deploy-mq
       taskRef:
-        name: helm-deploy-task-ace-api
+        name: build-push-docker-image
       params:
         - name: imageTag
           value: "$(params.imageTag)"
-    - name: ace-acme-helm-deploy
+        - name: imageName
+          value: "ace-chris"
+        - name: dockerfile
+          value: "ACE-Chris.Dockerfile"
+        - name: contextPath
+          value: "Bar_files/ace-chris"
+      resources:
+        inputs:
+          - name: git-source
+            resource: git-source
+    - name: deploy-ace-int-server-ace-api
       runAfter:
-        - ace-acme-build
+        - build-ace-int-server-ace-api
       taskRef:
-        name: helm-deploy-task-ace-acme
+        name: deploy-task
       params:
         - name: imageTag
           value: "$(params.imageTag)"
-    - name: ace-bernie-helm-deploy
+        - name: imageName
+          value: "ace-api"
+        - name: releaseName
+          value: "ace-api-int-srv-$(params.imageTag)"
```

Comment:
Use of imageTag in the releaseName applies here too and to the other ace deployments

Reply:
Done